pkg/builder/AppDetailBuilder.go

```diff
@@ -384,6 +384,130 @@ func GetDeploymentDetail(request *client.DeploymentDetailRequest) (*client.Deplo
 	return resp, nil
 }
```

Comment:
change file name to represent current functionality

Reply:
Will do this in my next item (Refactoring #1152)

Reply:
done